waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/WaggleDance.java

```diff
@@ -79,7 +79,9 @@ public static void main(String[] args) throws Exception {
       }
       throw e;
     }
-    System.exit(exitCode);
+    if (exitCode != 0) {
+      throw new Exception("Waggle didn't exit properly see logs for errors, exitCode=" + exitCode);
```

Comment:
Waggle -> Waggle Dance

Won't this now change the exit code into whatever a thrown exception causes the exit code to be? Maybe we don't care as long as we're sure this isn't 0?

Reply:
sorry